tests/manage/pv_services/test_write_to_cephfs_pvc_with_nonroot_user.py

```diff
@@ -0,0 +1,109 @@
+import logging
+
+from ocs_ci.ocs import constants
+from ocs_ci.helpers.helpers import create_pod
+from ocs_ci.framework.testlib import ManageTest, tier1, bugzilla, polarion_id
+from ocs_ci.ocs.resources.pod import run_io_in_bg
+from ocs_ci.ocs.exceptions import TimeoutExpiredError
+
+log = logging.getLogger(__name__)
+
+
+@tier1
+@bugzilla("2176354")
+@polarion_id("OCS-5139")
+class TestToWriteToCephfsPVCWithNonRootUser(ManageTest):
+    """
+    Test to write to cephfs PVC with nonRoot and fsGroup permissions
+    """
+
+    def test_to_write_to_cephfs_pvc_with_NonRootUser(
+        self,
+        multi_pvc_factory,
+        teardown_factory,
+    ):
+        """
+        Test Steps:
+        1. Create a pod with scc with runAsUser and runAsNonRoot
+        2. Perform IOs, should give permission error with both the cephfs access_modes
+        3. Recreate pod with fsGroup scc
+        4. Perform io, should be successful on the pods.
+
+        """
+        command = [
+            "sh",
+            "-c",
+            "echo The app is running! && sleep 3600",
+        ]
```

Comment:
why do we need this?

Reply:
Pod is expected to go CBLO with the default internal command of the ngnix image with scc, hence its necessary to pass commands to avoid pod reaching CBLO.